TutorialApplication/MainViewController.m

```diff
@@ -29,6 +29,10 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
+   [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(getNotification) name:@"faifly.alex.yegorov" object:nil];
```

Comment:
If you register for notifications you should always unregister from them in the according method.